code/modules/mob/status_procs.dm

```diff
@@ -110,6 +110,14 @@
 /mob/proc/set_drugginess(amount)
 	return
 
+/////////////////////////////////// HYGIENE ////////////////////////////////////
+
+/mob/proc/adjust_hygiene(amount)
```

Comment:
there's no justification for this, you don't even have anything outside of mob/living/carbon that this will work for

Reply:
🤔 yeah but every status effect uses it like this and every time I ask if this is what I should do they tell me whatever. have you even seen the defines on mob/living

Reply:
it's bullshit, nobody is ever going to move these features up to the mob level and they're just doing it to save themselves typing, it's lazy ass shit

Reply:
ok good well then now i finally know because everyone else gives me vague shit answers about it.